src/main/scala/org/deepdive/ddlog/DeepDiveLogDeltaDeriver.scala

```diff
@@ -1,5 +1,7 @@
 package org.deepdive.ddlog
```

Comment:
I'd insist on keeping it.  I saw the delta program derivation part was clean enough to keep around, and named it a separate command `print-incremental`.  The incremental support got all over the compiler because DeepDive didn't expose much control back then.  With this, we may be able to implement it more cleanly by taking the delta rules out of DDlog and doing everything from the DeepDive side.